librz/core/cmd_descs/cmd_debug.yaml

```diff
@@ -625,6 +625,48 @@ commands:
       - RZ_OUTPUT_MODE_QUIET
     default_mode: RZ_OUTPUT_MODE_STANDARD
     args: []
+  - name: dk
+    summary: Debug signals management
+    subcommands:
+      - name: dk
+        summary: Send signal to the child
+        cname: cmd_debug_signal
+        args:
+          - name: signal
+            type: RZ_CMD_ARG_TYPE_NUM
```

Comment:
if you use `rz_num_math(core->math, argv[x])` in the code this should be RZNUM because it will resolve flags etc.

Reply:
But it's a signal number, there is no need to resolve flags or anything like that.